src/ds-style-fix.scss

```diff
@@ -817,3 +817,25 @@ div.contact-button {
 .mat-drawer-container {
   z-index: unset !important;
 }
+
+.selected {
+  background-color: #7987a5;
+  color: #ffffff;
+  a {
+    color: #ffffff;
+  }
+}
+
+mat-list.org-accordion-list > mat-list-item {
+  height: 30px !important;
+  font-size: 12px !important;
+}
+
+mat-panel-title.org-accordion-header {
+  color: #487980;
+  font-size: 16px;
+}
+
+.register-workflow-modal-button {
+  margin-top: 10px !important;
```

Comment:
make it into a universal class name (mt-2)  and keep it in this file.  Otherwise move it to the component's scss file.

Reply:
Is there documentation on these naming conventions?

Reply:
not an official thing, but if the very common styling like padding, margin, I name it the same as the bootstrap ones: https://getbootstrap.com/docs/4.1/utilities/spacing/